partiql-parser/Cargo.toml

```diff
@@ -22,7 +22,7 @@ bench = false
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [build-dependencies]
-lalrpop = "0.19.8"
+lalrpop = "0.19.12"
```

Comment:
Honestly, this can probably just be "0.19"; I'm not sure why I specified down to the patch version when I initially brought this in.